Header header
float32 drone_timeuint16 tag
uint16 size
int32 vzimmuLSB
float32 vzfind
